src/pages/news/news-add/index.js

```diff
@@ -0,0 +1,3 @@
+import NewsAddPage from './news-add';
```

Comment:
Name of the component should match with the name of file and folder

Reply:
Redone!